.github/workflows/default.yml

```diff
@@ -14,20 +14,30 @@ jobs:
     runs-on: ubuntu-20.04
 
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2
+      - name: Load current commit
+        uses: actions/checkout@v3
         with:
           ref: ${{ github.ref }}
+      
+      - name: Check index.js rebuild
+        run: |
+          diff=$(git diff --name-only)
+          if [[ "${diff[*]}" == *"index.ts"* ]] && [[ "${diff[*]}" != *"index.js"* ]]; then
```

Comment:
Why are you checking only "index" files?

Reply:
corrected it